detekt-config.yml

```diff
@@ -67,7 +67,10 @@ complexity:
     threshold: 150
   LongMethod:
     active: true
-    threshold: 30
+    threshold: 20
```

Comment:
Just curious why do we revert it back to `20`? I recalled last time we intentionally updated it to `30` for this [issue](https://github.com/blyscuit/kmm-swiftUI-IC/pull/64#discussion_r1036680750), right? 🤔